### PR TITLE
remove deprecated feature: check files

### PR DESCRIPTION
**Pull request by ankushsonare1**

### CODE REVIEW

**Review by christophkloeffel — CHANGES_REQUESTED** [review]
Please also create a new entry in https://github.com/bmw-software-engineering/trlc/blob/main/CHANGELOG.md

File: trlc/parser.py [hunk 1]
```diff
@@ -1902,44 +1902,6 @@ def parse_rsl_file(self):
 
         return ok
 
-    def parse_check_file(self):
-        # lobster-trace: LRM.Check_File
```
Comment (christophkloeffel) [inline trlc/parser.py:1906]: If you remove traces to the LRM
Always check if the corresponding content needs to be removed too

**Comment by christophkloeffel** [comment]
Regarding the LRM: Since we completely remove the support of check files. Please also have a look at the tracing report of LOBSTER
Make sure that the new code matches the LRM

**Comment by christophkloeffel** [comment]
> Please also create a new entry in https://github.com/bmw-software-engineering/trlc/blob/main/CHANGELOG.md

This is still pending

File: language-reference-manual/lrm.trlc
Comment (christophkloeffel) [inline language-reference-manual/lrm.trlc:1]: Please also have a look at section 3 and adapt it accordingly

**Review by christophkloeffel — CHANGES_REQUESTED** [review]

**Review by akashsurwase1 — DISMISSED** [review]
ok to merge

**Comment by akashsurwase1** [comment]
ok to merge

**Comment by florianschanda** [comment]
Obsoleted by https://github.com/bmw-software-engineering/trlc/pull/87

**Pull request closed without merging**